test/test_fmmoperators.jl

```diff
@@ -0,0 +1,117 @@
+using BEAST
+using CompScienceMeshes
+using ExaFMMt
+using FastBEAST
+using IterativeSolvers
+using LinearAlgebra
+using MKL
+using StaticArrays
+using Test
+
+r = 10.0
+λ = 20 * r
+k = 2 * π / λ
+
+sphere = meshsphere(r, 0.1 * r)
+X0 = lagrangecxd0(sphere)
+X1 = lagrangec0d1(sphere)
+
+S = Helmholtz3D.singlelayer(; gamma=im * k)
+D = Helmholtz3D.doublelayer(; gamma=im * k)
+Dt = Helmholtz3D.doublelayer_transposed(; gamma=im * k)
+N = Helmholtz3D.hypersingular(; gamma=im * k)
+
+q = 100.0
+ϵ = 1.0
+
+# Interior problem
+# Formulations from Sauter and Schwab, Boundary Element Methods(2011), Chapter 3.4.1.1
+pos1 = SVector(r * 1.5, 0.0, 0.0)  # positioning of point charges
+pos2 = SVector(-r * 1.5, 0.0, 0.0)
+
+function Φ_inc(x)
```

Comment:
The monopole excitation is now part of BEAST and I recommend to use it. The unit test of BEAST has been changed, so you can use it as pattern.